test/src/module/integration/allTest.c

```diff
@@ -45,6 +45,10 @@ Test statuses
 
 #define TEST_RESTORE_POINT                                          "pgbackrest"
 
+#define UPDATE_SPOOL_PATH() \
+    hrnHostConfigUpdateP(   \
+        .archiveAsync = BOOL_TRUE_VAR, .spoolPath = VARSTR(strNewFmt("%s/" STRINGIFY(__COUNTER__), strZ(hrnHostSpoolPath(pg1)))))
```

Comment:
```suggestion

#define UPDATE_SPOOL_PATH(host) \
    hrnHostConfigUpdateP(   \
        .archiveAsync = BOOL_TRUE_VAR, .spoolPath = VARSTR(strNewFmt("%s/" STRINGIFY(__COUNTER__), strZ(hrnHostSpoolPath(host)))))
```

Reply:
This is not the best idea since `hrnHostConfigUpdateP` updates the config for all hosts at once. Instead, I made it so that we specify the postfix instead of the full path. Which is then added to the base path.